test/CMakeLists.txt

```diff
@@ -334,18 +335,22 @@ function(rate_added_test NAME)
     endif()
 endfunction()
 
-
-
 foreach(TEST ${TESTS})
     get_filename_component(BASE_NAME ${TEST} NAME_WE)
     add_test_executable(test_${BASE_NAME} ${TEST})
 endforeach()
 
-
-
 set_tests_properties(test_sqlite_perfdb test_perfdb
     PROPERTIES RUN_SERIAL On)
 
+if(MIOPEN_TEST_GFX1030)
+    if(WORKAROUND_ISSUE_1053 AND MIOPEN_TEST_ALL)
+        set_tests_properties(
+            test_soft_max
+            PROPERTIES RUN_SERIAL On)
+    endif()
+endif()
```

Comment:
Please move this just after line 277, thanks.

> PS Properties of executable tests should be set in in `add_test_executable`.

https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1168#discussion_r716100233

Reply:
Updated. 
I also searched and see some tests are "custom", are the rest all belong to here: ``add_test_executable``?

Reply:
Yes. I am going to eventually rename all custom tests from `test_*` to `custom_*` for clarity.